tools/run-qemu.py

```diff
@@ -29,8 +30,9 @@
             "user,id=net0,hostfwd=tcp:127.0.0.1:20022-:22,hostfwd=tcp:127.0.0.1:20080-:80",
             "-object",
             "filter-dump,id=fiter0,netdev=net0,file=virtio-net.pcap",
-            "-device",
```

Comment:
This line shouldn't be deleted. `isa-debug-exit ...` is the value for this option.